tests/Feature/Commands/MigrationMakeCommandTest.php

```diff
@@ -60,7 +60,7 @@
 
         $this->assertFalse(File::exists(base_path("database/migrations/$fileName")));
 
-        Artisan::call(command: 'application:migration UserPost');
+        Artisan::call(command: 'diamond:make:migration UserPost');
 
         $this->assertTrue(File::exists(base_path("database/migrations/$fileName")));
```

Comment:
Can you add `File::delete()` to delete the result of migration after run test command?

Reply:
aite